blockfrost-http-client/src/tests.rs

```diff
@@ -1,4 +1,5 @@
 use super::*;
+use crate::keys::{my_base_addr, TESTNET};
```

Comment:
I assume this just got left behind from something. Was this needed?

Reply:
It's used on line 58 and 73

Reply:
Huh. Okay. Weird. I guess since they're ignored they didn't get checked by the compiler!